aten/src/ATen/cuda/Exceptions.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <ATen/Error.h>
+
+#define AT_CUDNN_CHECK(STATUS)                                                 \
```

Comment:
"grumble grumble we shouldn't have this macro if cudnn is not enabled" (it's harmless enough though)